library/dokku_network_property.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from ansible.module_utils.basic import AnsibleModule
+import subprocess
+
+DOCUMENTATION = """
+---
+module: dokku_network_property
+short_description: Set or clear a network property for a given dokku application
+options:
+  global:
+    description:
+      - Whether to change the global network property
+    default: False
+    aliases: []
+  app:
+    description:
+      - The name of the app. This is required only if global is set to False.
+    required: True
+    default: null
+    aliases: []
+  property:
+    description:
+      - |
+        The network property to be be modified. This can be any property network
+        property supported by dokku (e.g., `initial-network`, `attach-post-create`,
+        `attach-post-deploy`, `bind-all-interfaces`, `static-web-listener`, `tld`).
+    required: True
+    default: null
+    aliases: []
+  value:
+    description:
+      - The value of the network property (leave empty to unset)
+    required: False
+    default: null
+    aliases: []
+author: Philipp Sessler
+requirements: [ ]
+"""
+
+EXAMPLES = """
+- name: Associates a network after a container is created but before it is started
+  dokku_network_property:
+    app: hello-world
+    property: attach-post-create
+    value: example-network
+
+- name: Associates the network at container creation
+  dokku_network_property:
+    app: hello-world
+    property: initial-network
+    value: example-network
+
+- name: Setting a global network property
+  dokku_network_property:
+    global: true
+    property: attach-post-create
+    value: example-network
+
+- name: Clearing a network property
+  dokku_network_property:
+    app: hello-world
+    property: attach-post-create
+"""
+
+
+def dokku_network_property_set(data):
+    is_error = True
+    has_changed = False
+    meta = {"present": False}
+
+    command = "dokku network:set {0} {1} {2}".format(
+        "--global" if data["global"] else data["app"],
+        data["property"],
+        data["value"] if "value" in data else "",
```

Comment:
`data["value"]` will only be executed by the engine if the condition `"value" in data` evalutes to true, otherwise the empty string value will be taken. I just tested this with a small python script and it should actually be fine.

Reply:
Ahh sorry, I just realized what you may have pointed at: The above solution works if `value` is not present at all. However, if `value`is set to `None` the expression actually evaluates to true. 

I have to fix that. Thanks for spotting that!